libxscrnsaver will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libxscrnsaver it will fail in the following way:

```
2024/11/12 23:12:57 INFO running step "test/pkgconf"
2024/11/12 23:12:57 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:57 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:57 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:57 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:57 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 WARN + basename /home/build/melange-out/libxscrnsaver-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 WARN + dev_pkg=libxscrnsaver-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 WARN + apk info -L libxscrnsaver-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 WARN + grep -q ^Name: usr/lib/pkgconfig/xscrnsaver.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 WARN + basename usr/lib/pkgconfig/xscrnsaver.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 WARN + lib_name=xscrnsaver uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 WARN + echo usr/lib/pkgconfig/xscrnsaver.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 WARN + grep -q '^usr/lib/pkgconfig/xscrnsaver.pc$\|^usr/share/pkgconfig/xscrnsaver.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 WARN + pkgconf --exists xscrnsaver uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 WARN + pkgconf --print-errors --exists xscrnsaver uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 WARN Package scrnsaverproto was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 WARN Perhaps you should add the directory containing `scrnsaverproto.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 WARN Package 'scrnsaverproto', required by 'xscrnsaver', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:57 INFO pod 22cf308a260987258d6e215f4676eec525c39d02a80d883e854f6f7b790f8b9b terminated
2024/11/12 23:12:57 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:57 INFO   workspace dir: /home/user/tmp/melange-workspace-1873177240
2024/11/12 23:12:57 INFO   guest dir: /home/user/tmp/melange-guest-2692438072
2024/11/12 23:12:57 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libxscrnsaver] Error 1
```

--- a/libxscrnsaver.yaml
+++ b/libxscrnsaver.yaml
@@ -1,7 +1,7 @@
 package:
   name: libxscrnsaver
   version: 1.2.4
-  epoch: 0
+  epoch: 1
   description: X.Org screen saver extension
   copyright:
     - license: MIT
@@ -39,8 +39,12 @@ subpackages:
       - uses: split/dev
     dependencies:
       runtime:
+        - libx11-dev
         - libxscrnsaver
     description: libxscrnsaver dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: libxscrnsaver-doc
     pipeline:

